travel_platform/travel_app/templates/travel_app/main.html

```diff
@@ -14,6 +14,7 @@
 
   <div class="col-md-4 choose">
       <div id="map" class="map"></div>
+      <script type="text/javascript" src='{% static "travel_app/js/map.js" %}'></script>
```

Comment:
This script doesn't work if it is imported in normal way.
In every example with OpenLayers I found the script was integrated in this way.
As you can see, all the rest js scripts added properly.